src/Backups/BackupsWorker.cpp

```diff
@@ -1157,6 +1157,8 @@ void BackupsWorker::waitAll()
     for (const auto & id : current_operations)
         wait(id, /* rethrow_exception= */ false);
 
+    backup_log->flush(backup_log->getLastLogIndex());
```

Comment:
There is some tests
`test_shutdown_cancel_backup` and `test_shutdown_wait_backup` they expects that before shutdown all the logs is flushed and after restart the tests see the result. 
I do not see where we do that in the code. Do we have such gurantee?

Reply:
We could do the flush only for backups logs or make it for all table at `SystemLogs::shutdown`.

Reply:
Will add flush for all logs at shutdown.